lib/ddr/managers/workflow_manager.rb

```diff
@@ -5,6 +5,17 @@ class WorkflowManager < Manager
       PUBLISHED = "published"
       UNPUBLISHED = "unpublished"
 
+      def publishable?
```

Comment:
As we discussed, please implement this method separately on the descendants of Ddr::Models::Base.  The condition on Item and Component could, I think, be:

```ruby
def publishable?
   parent.present? && parent.published?
end
```